tool/tb_bin.py

```diff
@@ -22,34 +22,30 @@
 import os
 from cPickle import load
```

Comment:
Do you need cPickle?  

Can this get converted to pickle so that the code is functional across py2 and py3

Reply:
Yes. There should be no problem.
Again, remember tadbit is not prepared for Python3

Reply:
For the moment in the docs tests I have added cPickle to the list of modules that are mocked so they are now passing.